src/index.js

```diff
@@ -1,11 +1,19 @@
+import './styles/starter-page-templates-editor.scss';
+
+import TemplateSelectorControl from './components/template-selector-control';
+
 ( function( wp ) {
     const registerPlugin = wp.plugins.registerPlugin;
     const createElement = wp.element.createElement;
     const { Modal, Button } = wp.components;
     const { withState } = wp.compose;
     
-    const insertTemplate = () => {
-        fetch('https://www.mocky.io/v2/5cd3fb89350000de307a5211')
+    const insertTemplate = template => {
```

Comment:
Why not a custom action using `controls` + `withDispatch`.

Reply:
Why not indeed! 😄 

We need to start to learn about such things and then we can start refactoring the implementation.

Thanks for the advice.

Reply:
This code is only a prototype and would be replaced by a proper solution once we connect to a real API soon in #6.

Reply:
Makes sense, I'm all for prototypes :)